From 94952a26388b79fcb45c1a0e99a06eca0ffdf01d Mon Sep 17 00:00:00 2001
From: Ilies CHERGUI <ichergui@nvidia.com>
Date: Tue, 7 May 2024 11:03:10 +0100
Subject: [PATCH 2/3] Fix builds without x11

Upstream-Status: Inappropriate [OE specific]

Signed-off-by: Kurt Kiefer <kurt.kiefer@arthrex.com>
Signed-off-by: Matt Madison <matt@madison.systems>
---
 ext/eglgles/gsteglglessink.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/ext/eglgles/gsteglglessink.c b/ext/eglgles/gsteglglessink.c
index 44eeba7..4e1139a 100644
--- a/ext/eglgles/gsteglglessink.c
+++ b/ext/eglgles/gsteglglessink.c
@@ -107,7 +107,9 @@
 #include <gst/video/gstvideopool.h>
 #include <gst/video/videooverlay.h>
 
+#ifdef USE_EGL_X11
 #include <X11/Xlib.h>
+#endif
 
 #include "gstegladaptation.h"
 #include "video_platform_wrapper.h"
-- 
2.34.1

